### ISSUE TITLE
Error in ERT when selecting parameter in the correlation viewer

### ISSUE THREAD

**Issue by oysteoh** [equinor/webviz-ert#223]
Using dark-storage, while clicking around in webviz-ert, i encountered a bug in ert when using the correlation viewer.

Steps to reproduce:
- Download latest drogon from `/scratch/fmu/share/drogon/course/drogon_current.tar.gz` ( currently `drogon_20211115_12-12` )
- Source `komodo bleeding`
- From `drogon_20211115_12-12/resmod/ff/21.0.1/ert/model` - run `ert vis drogon_ahm_noseis.ert` 
- Go to `Response Correlation Viewer`
- Choose the `pred_op5` ensemble
- Select a random response and then
- Click on the `GLOBVAR:VOLON_PERMH_CHANNEL` parameter

i'm getting this error in the log
```
INFO:     127.0.0.1:40098 - "GET /ensembles/0a247005-ae4c-4467-ad76-10aca0f3c544/records/GLOBVAR%3AVOLON_PERMH_CHANNEL HTTP/1.1" 200 OK
DEBUG:urllib3.connectionpool:http://127.0.0.1:51838 "GET /ensembles/0a247005-ae4c-4467-ad76-10aca0f3c544/records/GLOBVAR:VOLON_PERMH_CHANNEL HTTP/1.1" 200 2712
/prog/res/komodo/bleeding-py36-rhel7/root/lib64/python3.6/site-packages/scipy/stats/_distn_infrastructure.py:1760: RuntimeWarning:

invalid value encountered in true_divide
```

followed by 

```

Error message: double_vector_assert_index: index:36 invalid. Valid interval: [0,0>.
```

dumpfile looks like this 

```
[ooh@st-linrgsn225 ~]$ cat /tmp/ert_abort_dump.ooh.20220222-081132.log


Abort called from: double_vector_assert_index (/tmp/jenkins-komodo-f_komodo/workspace/x-build-some-other/bleeding-py36-rhel7/komodo-releases/cache/ecl-main/_skbuild/linux-x86_64-3.6/cmake-build/lib/double_vector.cpp:182)

Error message: double_vector_assert_index: index:36 invalid. Valid interval: [0,0>.


--------------------------------------------------------------------------------
Stack trace (most recent call last):
#31   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#30   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#29   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#28   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#27   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#26   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#25   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#24   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#23   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#22   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#21   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#20   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c817, in _PyEval_EvalFrameDefault
#19   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56d7e372, in _PyGen_Send
#18   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0b146, in _PyEval_EvalFrameDefault
#17   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e16b52, in PyEval_EvalFrameEx
#16   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e167e9, in PyEval_EvalFrameEx
#15   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e159a3, in PyEval_EvalFrameEx
#14   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0b146, in _PyEval_EvalFrameDefault
#13   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e16b52, in PyEval_EvalFrameEx
#12   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e167e9, in PyEval_EvalFrameEx
#11   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e159a3, in PyEval_EvalFrameEx
#10   Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0c2fc, in _PyEval_EvalFrameDefault
#9    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e16b52, in PyEval_EvalFrameEx
#8    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e167e9, in PyEval_EvalFrameEx
#7    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e15e0e, in PyEval_EvalFrameEx
#6    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e0b146, in _PyEval_EvalFrameDefault
#5    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56e16a2e, in PyEval_EvalFrameEx
#4    Object "/lib64/libpython3.6m.so.1.0", at 0x7fed56daa346, in _PyCFunction_FastCallDict
#3    Object "/prog/res/komodo/bleeding-py36-rhel7/root/lib64/python3.6/site-packages/res/_lib.cpython-36m-x86_64-linux-gnu.so", at 0x7fed466d2747, in PyInit__lib
#2    Object "/prog/res/komodo/bleeding-py36-rhel7/root/lib64/python3.6/site-packages/res/_lib.cpython-36m-x86_64-linux-gnu.so", at 0x7fed466c62ef, in conf_instance_get_path_error
#1    Object "/prog/res/komodo/bleeding-py36-rhel7/root/lib64/python3.6/site-packages/ecl/.libs/libecl.so", at 0x7fed46ab3bab, in double_vector_iget
#0    Object "/prog/res/komodo/bleeding-py36-rhel7/root/lib64/python3.6/site-packages/ecl/.libs/libecl.so", at 0x7fed46a9acbe, in util_abort__
--------------------------------------------------------------------------------
```

The error is leaving `webviz-ert` in a state where it does not operate anymore and needs to be restarted

**Comment by frode-aarstad** [comment]
This is reproducible in the qtplotter. 

It is also a problem in older komodo releases. I tried on 2021-11-b8

**Comment by oysteoh** [comment]
@jcrivenaes and / or maybe you @berland ? - do you have any input to this issue? While we was "clicking around" in the Drogon case as specified in the description of this issue ( not only does this crash in webviz-ert - it also crashing the ERT GUI if clicked. ) we encountered this bug ( which we also have fixed in #2969 ). But - is it faulty data produced in the Drogon case, or is it actually a bug in ERT? If it is a bug we want to write some tests targeting this piece of code to verify exactly what could go wrong and how we should fix it. Should we inform the users with some logging statements ?

**Comment by oysteoh** [comment]
I've got some information. The reason the parameter crashes for `pred_*` in this case is that it is not present at all. Not all parameters are available on all the ensembles. So - i guess this is functionality which should be discussed wrt `webviz-ert`. Should we at any given time only show parameters available in all selected ensembles?
